src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,32 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RegistrationException("User is null");
+        }
+        int age = user.getAge() == null ? 0 : user.getAge();
+        String login = user.getLogin() == null ? null : user.getLogin();
+        String password = user.getPassword() == null ? null : user.getPassword();
```

Comment:
not like this, remove and rewrite like:
if (fieldIsInvalid) {
throw some exception
}